src/vm/comutilnative.cpp

```diff
@@ -1524,7 +1528,11 @@ FCIMPL5(VOID, Buffer::InternalBlockCopy, ArrayBase *src, int srcOffset, ArrayBas
     _ASSERTE(count >= 0);
 
     // Copy the data.
+#if defined(_WIN64)  
+    JIT_MemCpy(dst->GetDataPtr() + dstOffset, src->GetDataPtr() + srcOffset, count);
+#else    
```

Comment:
There are number of other uses of memmove. If this is faster than the stock memmove implementation, should it be re-defined in some more central header instead?